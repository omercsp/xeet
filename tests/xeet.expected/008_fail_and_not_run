Starting run
Running tests: 002_fail, 004_bad_desc

Running 002_fail                                             .......[Failed ] rc=1, allowed=2
Full output file: XEET_DIR/sanity/xeet.out/002_fail

Running 004_bad_desc                                         .......[Not Run]
Schema validation error: Additional properties are not allowed ('blat' was unexpected)


Finished iteration (#0/0)
Failed: 002_fail
Not ran: 004_bad_desc

