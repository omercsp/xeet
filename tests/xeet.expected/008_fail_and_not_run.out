Starting run
Running tests: 002_fail, 004_bad_desc

002_fail                                                     ....... [Failed ] rc=1, allowed=0
Test stdout/stderr: (Empty stdout), (Empty stderr)

004_bad_desc                                                 ....... [Not Run]
Schema validation error: Additional properties are not allowed ('blat' was unexpected)


Finished iteration (#0/0)
Failed: 002_fail
Not ran: 004_bad_desc

